t/lang.t

```diff
@@ -78,4 +78,24 @@ sub test_logo_exists {
 test_logo_exists('logo');
 test_logo_exists('logo2x');
 
+# Chech that some words like Eco-Score and Nutri-Score are not translated
+
+my @words_that_should_not_be_translated = ("Nutri-Score", "Eco-Score", "NOVA");
+
+foreach my $stringid (sort keys %Lang) {
+	foreach my $word (@words_that_should_not_be_translated) {
+		if ($Lang{$stringid}{'en'} =~ /\b$word\b/) {
```

Comment:
I didn't notice how much it took to run this, but using a regexp with all words and matching once, should be roughly 3x faster.

`/\b(Nutri-Score|Eco-Score|NOVA)\b` 

I imagine that checking the same regexp in translation (whithout searching which one it is), is engouh, but if needed, we can do an inner loop to then check each words.